avionics/common/lib/CalcHelpers/barometric.cpp

```diff
@@ -6,13 +6,15 @@
  * @param pascals Pressure, in [Pa]
  * @return Altitude above sea level [m]
  */
-float pressureToAltitude(float const pres) {
+float /* m */ pressureToAltitude(float pres /* mbar */) {
```

Comment:
there's a documentation section right above the function that lists units, and also commit 8da79407 updates the barometer sensor object to return Pa instead (my bad, I should've updated you beforehand)

note this also means below changes to tests should be reverted

Reply:
So with those changes we use Pa everywhere? So should be changed in the GS/SIM as well?

Reply:
yeah, probably - it's probably best to use SI base units wherever possible